howdoi/howdoi.py

```diff
@@ -643,9 +648,9 @@ def _sanity_check(test_query=None):
     error_result = b"Sorry, couldn't find any help with that topic\n"
 
     if _clear_cache():
-        _print_ok('Cache cleared successfully')
+        print(f'{GREEN}Cache cleared successfully')
```

Comment:
When using `GREEN` you must also use `END_FORMAT`.

For example right now this entire query is green:

```
python -m howdoi.howdoi format date bash -C
```